common/config/rush/pnpm-lock.yaml

```diff
@@ -1205,27 +1205,18 @@ packages:
       - supports-color
     dev: false
 
-  /@azure/msal-common/5.2.0:
```

Comment:
Do we need to rever @azure/msal-common 5.2.0 also?

Reply:
don't think so, only msal-node-extensions carries the dependency that we want to eliminate and the tests are looking good so far